lib/components/persistent_tab_view.dart

```diff
@@ -430,6 +430,7 @@ class _PersistentTabViewState extends State<PersistentTabView> {
       widget.handleAndroidBackButtonPress &&
       widget.onWillPop == null &&
       _controller.isOnInitialTab() &&
+      _navigatorKeys[_controller.index].currentState != null &&
```

Comment:
A small comment like "Required if historyLength == 0 because isOnInitialTab() is already true when switching to uninitialized tabs instead of only when going back" may be good for understanding why this is needed later.